docs/docs/tutorial.md

```diff
@@ -91,33 +83,33 @@ $ more mesos-dns.json
 "mem": 1024,
 "id": "mesos-dns",
 "instances": 1,
-"constraints": [["hostname", "CLUSTER", "10.14.245.208"]]
+"constraints": [["hostname", "CLUSTER", "2.3.4.5"]]
 }
 
 ```
 
-Launch Mesos-DNS using:
+Launch Mesos-DNS via Marathon using:
 
 ```
-curl -X POST -H "Content-Type: application/json" http://10.41.40.151:8080/v2/apps -d@mesos-dns.json
+curl -X POST -H "Content-Type: application/json" http://1.2.3.4:8080/v2/apps -d@mesos-dns.json
 ```
 
-This command instructs Marathon to launch Mesos-DNS on node `10.14.245.208`. The `-v` option allows you to capture detailed warning/error logs for Mesos-DNS that may be useful for debugging. For example, Mesos-DNS will now periodically print in `stdout` information about all the A and SRV records it servers. This can be useful if you are not sure about how tasks and frameworks are named in your setup. You can access the `stdout` and `stderr` for Mesos-DNS through the Mesos webUI, accessible through `http://10.41.40.151:5050` in this example. 
+This command instructs Marathon to launch Mesos-DNS on node `2.3.4.5`. The `-v` option allows you to capture detailed warning/error logs for Mesos-DNS that may be useful for debugging. For example, Mesos-DNS will now periodically print in `stdout` information about all the A and SRV records it servers. This can be useful if you are not sure about how tasks and frameworks are named in your setup. You can access the `stdout` and `stderr` for Mesos-DNS through the Mesos webUI, accessible through `http://1.2.3.4:5050` in this example. 
```

Comment:
Most of this paragraph doesn't make sense, starting with "The `-v` option...". It starts making more sense at "You can access..."